MyGame/MyGame/GameScene.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  GameScene.swift
+//  MyGame
+//
+//  Created by Sacchitiello Fabio on 2/28/18.
+//  Copyright © 2018 Sacchitiello Fabio. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    
+    //store score and display labels
+    var isGameStarted = Bool (false)
+    var isDead = Bool (false)
+    let collectSound = SKAction.playSoundFileNamed("CoinSound.mp3", waitForCompletion: false)
+    
+    var score = Int(0)
+    
+    var scoreLabel = SKLabelNode()
+    var highscoreLabel = SKLabelNode()
+    var playLabel = SKLabelNode()
+    
+    var restartButton = SKSpriteNode()
+    var pauseButton = SKSpriteNode()
+    var logoImg = SKSpriteNode()
+    
+    var wallBlocks = SKNode()
+    var moveAndRemove = SKAction()
+    
+    //Chopper atlas for animation
+    //let chopperAtlas = SKTextureAtlas(named: "player")
+    var chopperSprites: [SKTexture] = [SKTexture(imageNamed: "img1"), SKTexture(imageNamed: "img2"), SKTexture(imageNamed: "img3"), SKTexture(imageNamed: "img4")]
+    var chopper = SKSpriteNode.init(imageNamed: "img1")
+    var repeatActionChopper = SKAction()
+    
+    override func didMove(to view: SKView) {
+        // Get label node from scene and store it for use later
+        createScene()
+        
+        self.chopper = createChopper()
+        self.addChild(chopper)
+        //animate and repeat the chopper animation
+        let animateChopper = SKAction.animate(with: self.chopperSprites, timePerFrame: 0.1)
```

Comment:
All these details like setting up the choppers animation should be hidden from the game scene.  Think about it like you create a chopper, and it just behaves like a chopper should to the game scene, the scene shouldn't be setting up everything.